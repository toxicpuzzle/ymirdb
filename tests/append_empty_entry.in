SET a 1 
POP a
APPEND a 3 2 1 
LIST ENTRIES
bye