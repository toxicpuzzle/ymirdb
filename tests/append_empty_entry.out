> ok

> 1

> ok

> a [3 2 1]

> bye
